components/LPRegistrationForm.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <v-form
+    ref="form"
+    class="lp-registration-form w-full"
+    @submit.prevent="submitHandler"
+  >
+    <v-text-field
+      v-model="form.name"
+      label="ПІБ"
```

Comment:
Please let's use i18n and English version for now

Reply:
Done